lib/highline/import.rb

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env ruby
```

Comment:
Hey, no worries! Really good job for pushing it forward like this.

It was more like really honest question than a nitpick. I see it in some other files too. So I was thinking if it is on purpose to achieve some use case I don't know.

Will do a commit and reference it in the comments you you can just cherry-pick it.